corehq/apps/users/models.py

```diff
@@ -1410,7 +1410,11 @@ def get_by_username(cls, username, strict=False):
             include_docs=True,
             reduce=False,
         )
-        result = view_result.one()
+        result = view_result.all()
+        if len(result) > 1:
+            raise MultipleResultsFound('"{}": {}'.format(
+                username, ', '.join([row['id'] for row in result])
+            ))
         if result and result['doc'] and result['doc']['username'] == username:
```

Comment:
do you need to do something like `result[0]` now? I'm not super familiar with couchdbkit, but I imagine that `all` returns a list.

Reply:
oops